Carbamazepine-Model.json

```diff
@@ -35799,9 +26098,10 @@
                 "MolWeight": 236.269
               },
               "Values": [
-                0.00157480314
+                0.157480314
               ],
-              "Dimension": "Fraction"
+              "Dimension": "Fraction",
+              "Unit": "%"

```

Comment:
@AndreDlm 
Changed the units of Fraction SD from "" to "%" (and adjusted the values) to overcome the glitch in RE
(s. https://github.com/ErikssonJ/Rosuvastatin-Model/pull/1/files?w=1#r1967834202 for details)